app/models/invitation_to_organization.rb

```diff
@@ -0,0 +1,50 @@
+class InvitationToOrganization < Invitation
+  belongs_to :member, inverse_of: :created_invitations, foreign_key: :invited_by_id
+  belongs_to :organization, inverse_of: :invitations
+
+  validates :invited_by_id, :role, presence: true
+  validate :role_inclusion
+  validate :email_uniq
+
+  delegate :organization, to: :member
+
+  after_create :send_invitation
+
+  enumerize :role, in: [:user, :admin, :owner], default: :user, predicates: true
+
+  def owner
+    Member.find(self.invited_by_id).user.to_s
+  rescue
+    ''
+  end
+
+  def accept!(user)
+    user_member = Member.find_or_create_by!(user: user, organization_id: member.organization_id)
+    user_member.update_attribute(:role, role)
```

Comment:
@dissident this two strings generate two requests to db. Instead you may do it with one:
```rb
Member.find_or_initialize_by!(user: user, organization_id: member.organization_id).tap do |member|
  member.role = role
  member.save
end
```